src/django/api/views.py

```diff
@@ -1219,3 +1220,13 @@ def reject_match(self, request, pk=None):
             raise NotFound()
         except FacilityMatch.DoesNotExist:
             raise NotFound()
+
+
+@api_view(['GET'])
+@permission_classes((AllowAny,))
+def api_feature_flags(request):
+    response_data = {
+        'claim_a_facility': switch_is_active('claim_a_facility'),
```

Comment:
We should consider codifying the deployment-time task of creating this flag in the database by adding a data migration that creates the inactive flag record.

Reply:
Sounds good -- I created the migration here:

8303bd8a93bea2f2161c569134e46104fb566e83